projects/hslayers/src/components/add-data/url/add-data-url.service.ts

```diff
@@ -25,7 +27,8 @@ export class HsAddDataUrlService {
   constructor(
     public hsLog: HsLogService,
     public hsLanguageService: HsLanguageService,
-    public hsLayoutService: HsLayoutService
+    public hsLayoutService: HsLayoutService,
+    private hsMapsService: HsMapService
```

Comment:
```suggestion
    private hsMapService: HsMapService
```
With code-completion, this is not that much of a issue, but could be misleading..